d3d9ex/AutoFix.cpp

```diff
@@ -9,20 +8,21 @@
 void MainContext::EnableAutoFix()
 {
 	std::string exe_name = ModuleNameA(NULL);
-	std::transform(exe_name.begin(), exe_name.end(), exe_name.begin(), std::tolower);
+	//workaround for template deduction after including <locale>
+	std::transform(exe_name.begin(), exe_name.end(), exe_name.begin(), [&](int i) { return std::tolower(i); });

```

Comment:
template deduction fails here because <locale> also include templatized isspace and it's pulled with spdlog